graphql/server.go

```diff
@@ -377,6 +392,7 @@ func (c *conn) handleMutate(in *inEnvelope) error {
 
 		go c.rerunSubscriptionsImmediately()
 
+		initial = false
```

Comment:
i don't think this variable is necessary. mutations never re-run, so they will never have a non-initial computation